markdown/plext/using_pljava.html.md.erb

```diff
@@ -2,205 +2,222 @@
 
-2. Perform the following step on all machines to set up `ldconfig` for the installed JDK:
+2. Perform the following step on all nodes in your HAWQ cluster to set up the shared library cache for the installed JDK:
 
 	``` shell
-	$ echo "$JAVA_HOME/jre/lib/amd64/server" > /etc/ld.so.conf.d/libjdk.conf
-	$ ldconfig
+	root@hawq-node$ echo "$JAVA_HOME/jre/lib/amd64/server" > /etc/ld.so.conf.d/libjdk.conf
+	root@hawq-node$ ldconfig
 	```
 4. Make sure that your HAWQ cluster is running, you have sourced `greenplum_path.sh` and that your `$GPHOME` environment variable is set.
 
+    ``` shell
+	$ ssh gpadmin@master
+	gpadmin@master$ source /usr/local/hawq/greenplum_path.sh
+	gpadmin@master$ hawq state
+	```
 
 ### <a id="enablepljava"></a>Enable PL/Java and Install JAR Files 
 
-To use PL/Java:
+To use PL/Java, you must:
 
 1. Enable the language for each database.
-1. Install user-created JAR files on all HAWQ hosts.
-1. Add the names of the JAR files to the HAWQ `pljava_classpath` server configuration parameter. This parameter value should identify a list of the installed JAR files.
+1. Install custom JAR files on all HAWQ hosts.
+1. Add the names of the JAR files to the HAWQ `pljava_classpath` server configuration parameter. This parameter value should identify a list of all installed JAR files.
 
 #### <a id="enablepljava"></a>Enable PL/Java and Install JAR Files 
 
 Perform the following steps as the `gpadmin` user:
 
-1. Enable PL/Java by running the `$GPHOME/share/postgresql/pljava/install.sql` SQL script in the databases that will use PL/Java. The `install.sql` script registers both the trusted and untrusted PL/Java languages. For example, the following command enables PL/Java on a database named `testdb`:
+1. Enable PL/Java by running the `$GPHOME/share/postgresql/pljava/install.sql` SQL script on the databases that will use PL/Java. The `install.sql` script registers both the trusted (`java`) and untrusted PL/Java (`javau`) languages. For example, the following command enables the PL/Java languages on a database named `testdb`:
 
 	``` shell
-	$ psql -d testdb -f $GPHOME/share/postgresql/pljava/install.sql
+	gpadmin@master$ psql -d testdb -f $GPHOME/share/postgresql/pljava/install.sql
 	```
 	
 	To enable the PL/Java extension in all new HAWQ databases, run the script on the `template1` database: 
 
     ``` shell
-    $ psql -d template1 -f $GPHOME/share/postgresql/pljava/install.sql
+    gpadmin@master$ psql -d template1 -f $GPHOME/share/postgresql/pljava/install.sql
     ```
 
-    Use this option *only* if you are certain you want to enable PL/Java in all new databases.
+    Use this option *only* if you are certain you want to enable both the trusted and untrusted versions of PL/Java in all newly-created databases. This operation takes effect on all databases created after the command is invoked.
```

Comment:
The last sentence here can be read with two different meanings.  Let's make something like:  "After you invoke this command, PL/Java will be enabled in each new database that a user creates."